oci/private/tarball_run.sh.tpl

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+readonly IMAGE="{{image_path}}"
+docker load --input "$IMAGE"
```

Comment:
I would like this to check if a `docker` command is on the PATH and automatically fallback to `podman` (basically do a convenience search for installed runtimes)
BUT that's possible without a breaking API change as a follow-up PR so I'm happy to get this in as-is.

Reply:
Changed the script to fallback to podman. 

Do you want me to make `run_template` private?